packages/tasks/src/tasks/text-to-3d/about.md

```diff
@@ -0,0 +1,22 @@
+## Use Cases
+
+Text-to-3D models can be used in a wide variety applications that require 3D, such as games, animation, design, architecture, engineering, marketing, and more.
+
+This task is similar to the [image-to-3d](https://huggingface.co/tasks/image-to-3d) task, but takes text input instead of image input. In practice, this is often equivalent to a combination of [text-to-image](https://huggingface.co/tasks/text-to-image) and [image-to-3d](https://huggingface.co/tasks/image-to-3d).
```

Comment:
`In practice, this is often equivalent to a combination of [text-to-image](https://huggingface.co/tasks/text-to-image) and [image-to-3d](https://huggingface.co/tasks/image-to-3d).` how, as in, does it input image and text simultaneously?

Reply:
Can you clarify?